scripts/setup-env.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable consistent-return, no-console */
+
+'use strict';
+
+const fs = require('fs');
+const { execSync } = require('child_process');
+const path = require('path');
+
+function executeCommand(command) {
+  console.log(`Executing command ${command}`);
+  execSync(command);
+}
+
+function copyConfig() {
+  let command1;
+  let command2;
+
+  if (process.platform === 'Win32') {
+    command1 = 'copy okta-hosted-login/util/default-config.js okta-hosted-login/src/.samples.config.js';
+    command2 = 'copy custom-login/util/default-config.js custom-login/src/.samples.config.js';
+  } else {
+    command1 = 'cp okta-hosted-login/util/default-config.js okta-hosted-login/src/.samples.config.js';
+    command2 = 'cp custom-login/util/default-config.js custom-login/src/.samples.config.js';
+  }
+
+  console.log('Copying the configuration files...');
+  executeCommand(command1);
+  executeCommand(command2);
+}
+
+function updateConfig(file) {
+  if (process.env.ISSUER === undefined || process.env.CLIENT_ID === undefined) {
+    console.log('[ERROR] Please set the ISSUER and CLIENT_ID Environment variables');
```

Comment:
This should be console.error, and we should then `process.exit(1)` so that the next tasks won't run.